core/alsp_src/tests/libtests/rows_cols_test.pro

```diff
@@ -0,0 +1,103 @@
+:- [test].
+
+test_cols :-
+	assert(silly_list( 
+		[
+        	  [abc, rjtir, x, y],
+        	  [yrtbv, h8t, x, yacc],
+        	  [34, f(4,5,6), some_silly(55,yurht, 123), thru]
+        	] )  ),
+	test([
+	test_var_columns_file,
+	test_var_columns_file_uh,
+	test_fxd_columns_file, 
+	test_fxd_columns_file_uh,
+	remove_file('silly.txt'),
+	true
+	]),
+	abolish(silly_list/1).
+
+test_var_columns_file :-
+	silly_list(ListOfRows),
+	open('silly.txt', write, S),
+	columns(ListOfRows, S),
+	close(S),
+	grab_lines('silly.txt', SillyColLines),
+	T1 = 'abc    rjtir     x                         y     ',
+	'$strlen'(T1, T1Len),
```

Comment:
Just ‘cause I used ’$strlen' everywhere :-).  Old code of mine uses that.  No other reason.  Better to change to atom_length since it’s more modern.   FIXED

Reply:
Possibly the fix didn't take. I still only see `$strlen`...